cds_migrator_kit/rdm/migration/transform/user_transform.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# CDS-RDM is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+"""CDS-RDM transform user."""
+
+from datetime import datetime
+
+from invenio_rdm_migrator.streams.users import UserEntry, UserTransform
+
+
+class CDSUserTransform(UserTransform):
+    """CDSUserTransform."""
+
+    def _user(self, entry):
+        """Transform the user."""
+        return CDSUserEntry().transform(entry)
+
+    def _session_activity(self, entry):
+        """Transform the session activity."""
+        return entry.get("session_activity")
+
+    def _tokens(self, entry):
+        """Transform the tokens."""
+        return entry.get("tokens")
+
+    def _applications(self, entry):
+        """Transform the applications."""
+        # TODO
+        pass
+
+    def _oauth(self, entry):
+        """Transform the OAuth accounts."""
+        # TODO
+        pass
+
+    def _identities(self, entry):
+        """Transform the identities."""
+        data = entry.get("identities")
+        return [
+            {
+                "id": i["id"],
+                "created": i["created"],
+                "updated": i["updated"],
+                "method": i["method"],
+            }
+            for i in data or []
+        ]
+
+
+class CDSUserEntry(UserEntry):
```

Comment:
I think we should remove all the users part?

Reply:
I left it for now, because I have a vague thought we had implemented it there for a reason initially. Of course I don't remember the reason 🫠